WEBPACK_UPDATE.md

```diff
@@ -0,0 +1,50 @@
+# Upgrading from angular-cli@1.0.0-beta.10 to the angular-cli@webpack 
+
+To update `angular-cli` to the webpack preview, you must update both the global package, project's local package and your project files.
+
+## Global package:
+```
+npm uninstall -g angular-cli
+npm cache clean
+npm install -g angular-cli@webpack
+```
+
+## Local project package:
+```
+rm -rf node_modules dist tmp typings
+npm install --save-dev angular-cli@webpack
+```
+
+## Project files
+
+You will need to run `ng init` to check for changes in all the auto-generated files created by `ng new` and allow you to update yours. You are offered four choices for each changed file: `y` (overwrite), `n` (don't overwrite), `d` (show diff between your file and the updated file) and `h` (help).
+
+Carefully read the diffs for each code file, and either accept the changes or incorporate them manually after `ng init` finishes.
+
+You can find a sample github diff of the changes introduced between a beta.10 and webpack preview project at https://github.com/filipesilva/angular-cli-webpack-upgrade/commit/HEAD.
+
+Here is a summary of the file changes:
+
+1. Updated files:
+  * `./config/karma.conf.js` - `frameworks`/`plugins`/`files`/`exclude`/`preprocessors` entries changed, added `angularCliConfig` entry.
+  * `./e2e/tsconfig.json` - property cleanup, changed `outdir`, added `typeRoots`
+  * `./package.json` - removed typings postinstall script, removed `ember-cli-inject-live-reload`/`es6-shim`/`systemjs` dependencies, added `core-js`/`ts-helpers` dependencies, updated `angular-cli`/`codelyzer`/`ts-node`/`ts-lint`/`typescript` devDependencies, added `@types/jasmine`/`@types/protractor`/`karma-coverage` devDependencies.
```

Comment:
Should https://github.com/angular/angular-cli/pull/1468 be merged in before the `@webpack` release, `karma-remap-istanbul` will be added to the `devDependencies` list.